perception/lidar_transfusion/README.md

```diff
@@ -0,0 +1,128 @@
+# lidar_transfusion
+
+## Purpose
+
+The `lidar_transfusion` package is used for 3D object detection based on lidar data (x, y, z, intensity).
+
+## Inner-workings / Algorithms
+
+The implementation bases on TransFusion [1] work. It uses TensorRT library for data process and network inference.
+
+We trained the models using <https://github.com/open-mmlab/mmdetection3d>.
+
+## Inputs / Outputs
+
+### Input
+
+| Name                 | Type                            | Description       |
+| -------------------- | ------------------------------- | ----------------- |
+| `~/input/pointcloud` | `sensor_msgs::msg::PointCloud2` | Input pointcloud. |
+
+### Output
+
+| Name                                   | Type                                                  | Description                 |
+| -------------------------------------- | ----------------------------------------------------- | --------------------------- |
+| `~/output/objects`                     | `autoware_auto_perception_msgs::msg::DetectedObjects` | Detected objects.           |
+| `debug/cyclic_time_ms`                 | `tier4_debug_msgs::msg::Float64Stamped`               | Cyclic time (ms).           |
+| `debug/pipeline_latency_ms`            | `tier4_debug_msgs::msg::Float64Stamped`               | Pipeline latency time (ms). |
+| `debug/processing_time/preprocess_ms`  | `tier4_debug_msgs::msg::Float64Stamped`               | Preprocess (ms).            |
+| `debug/processing_time/inference_ms`   | `tier4_debug_msgs::msg::Float64Stamped`               | Inference time (ms).        |
+| `debug/processing_time/postprocess_ms` | `tier4_debug_msgs::msg::Float64Stamped`               | Postprocess time (ms).      |
+| `debug/processing_time/total_ms`       | `tier4_debug_msgs::msg::Float64Stamped`               | Total processing time (ms). |
+
+## Parameters
+
+### Core Parameters
+
+| Name                             | Type         | Default Value | Description                                                                                        |
+| -------------------------------- | ------------ | ------------- | -------------------------------------------------------------------------------------------------- |
+| `class_names`                    | list[string] | -             | Class names for 3D object detection.                                                               |
```

Comment:
@amadeuszsz 
Sorry for the late comment, but since you added the schema file, could you change the README.md so it uses the schema instead?

Reply:
Done 👍🏻